pom.xml

```diff
@@ -31,11 +31,11 @@
         <organization>Salesforce</organization>
     </developer>
     <developer>
-        <name>Steven Tamm</name>
+        <name>Andrzej Poblocki</name>
         <roles>
             <role>Member</role>
         </roles>
-        <email>stamm@salesforce.com </email>
+        <email>apoblock@salesforce.com </email>
```

Comment:
@yoikawa, I don't feel I deserve any credit here, but my email is `apoblocki@salesforce.com`